flaskProject1/static/js/Register/Register1.js

```diff
@@ -5,6 +5,7 @@ class Register1 extends React.Component {
 
         }
     }
+    
```

Comment:
You must avoid  unnecessary blank lines !

Reply:
Ok boss! I'll do my best!